packages/form-config/src/components/Releases/tests/__snapshots__/index.test.tsx.snap

```diff
@@ -53,6 +53,7 @@ Object {
       "updatedAt": "2020-06-08T15:00:38.413+02:00",
     },
   ],
+  "drillDownFilter": [Function],
```

Comment:
what causes this to change

Reply:
As discussed earlier, seems this was caused by antd minor version updates